aiohttp_jinja2/__init__.py

```diff
@@ -16,8 +16,10 @@
 APP_CONTEXT_PROCESSORS_KEY = 'aiohttp_jinja2_context_processors'
 
 
-def setup(app, *args, app_key=APP_KEY, context_processors=(), **kwargs):
+def setup(app, *args, app_key=APP_KEY, context_processors=(), filters=None, **kwargs):
     env = jinja2.Environment(*args, **kwargs)
+    if filters and isinstance(filters, dict):
```

Comment:
`isinstance` check is redundant: passing a list of filters leads to silent parameter ignorance which is confusing.